packages/auth/test/integration/flows/totp.test.ts

```diff
@@ -138,11 +136,15 @@ describe(' Integration tests: Mfa TOTP', () => {
       resolver = getMultiFactorResolver(auth, error as any);
       expect(resolver.hints).to.have.length(1);
 
+      totpTimestamp.setSeconds(totpTimestamp.getSeconds() + 30);
```

Comment:
nit: add a comment : "fetch the OTP code for the next window, to avoid reusing OTP code."